draft-ietf-quic-transport.md

```diff
@@ -2511,11 +2531,10 @@ mentions a stream ID.  In this state, the endpoint MUST observe advertised
 stream and connection data limits (see {{flow-control}}).
 
 A stream transitions from this state to "closed" by completing transmission of
-all data.  This includes sending all data carried in STREAM frames up including
+all data.  This includes sending all data carried in STREAM frames including
 the terminal STREAM frame that contains a FIN flag.
 
-A stream becomes "closed" when the endpoint sends and receives acknowledgment of
-a RST_STREAM frame.
+A stream also becomes "closed" when the endpoint sends a RST_STREAM frame.
```

Comment:
Hm, that's probably a change that came after my PR forked and I lost in the merge.  But I thought the transition to "closed" on the sender was immediate on sending the frame? I'll have to look at history and see if I can find when that changed.